java-manta-client/src/test/java/com/joyent/manta/client/MantaHttpHeadersTest.java

```diff
@@ -0,0 +1,33 @@
+package com.joyent.manta.client;
+
+import junit.framework.Assert;
```

Comment:
Oops, I'm assuming this is an incorrect import.  should be testng package instead of junit